src/main/java/uk/gov/hmcts/cmc/claimstore/controllers/DocumentsController.java

```diff
@@ -57,7 +57,7 @@ public DocumentsController(DocumentsService documentsService) {
         @PathVariable("externalId") @NotBlank String externalId,
         @RequestHeader(HttpHeaders.AUTHORIZATION) String authorisation
     ) {
-        byte[] pdfDocument = documentsService.getLegalSealedClaim(externalId, authorisation);
+        byte[] pdfDocument = documentsService.getSealedClaim(externalId, authorisation);
```

Comment:
That is breaking change. Legal sealed copy is different and should not get changed